src/index.ts

```diff
@@ -49,11 +49,13 @@ export class Chatty {
    * for details.
    *
    * @param url The URL of the client iframe to create. The hosted iframe should create a chatty
-   * client to communicate with the host.
+   * client to communicate with the host. Optional, if not provided, source will be used,
+   * @param source The source of the client iframe to create. The hosted iframe should create a chatty
+   * client to communicate with the host. Otional, if not provided, url will be used.
    */
 
-  static createHost (url: string) {
-    return new ChattyHostBuilder(url)
+  static createHost (url?: string, source?: string) {
+    return new ChattyHostBuilder(url, source)
```

Comment:
I might prefer a second `createHostFromSource()` method than one with an "optional" null parameter.

Reply:
Dammit, thats what I did originally :D 

I will change it.

Thanks